packages/patternfly-3/package.json

```diff
@@ -4,40 +4,6 @@
   "config": {
     "access": "private"
   },
-  "devDependencies": {
```

Comment:
These are ignored by yarn since the package has no version, and there's nothing we can do about it :(